app.rb

```diff
@@ -0,0 +1,114 @@
+module App
+  INPT_MSG = 'Enter an option number here --> '.freeze
+  ENTR_MSG = 'Press ENTER to continue'.freeze
+
+  def enter_msg
+    print ENTR_MSG
+    gets
+    puts
+  end
+
+  def display_books
+    puts 'List of books:'
+    puts 'EMPTY' if @books.empty?
+    @books.each_with_index { |book, index| puts "#{index}) Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def display_people
+    puts 'List of people:'
+    puts 'EMPTY' if @people.empty?
+    @people.each_with_index do |person, index|
+      puts "#{index}) Name: #{person.name}, Age: #{person.age}, ID: #{person.id}"
+    end
+  end
+
+  # rubocop:disable Metrics
+  def create_person
+    decision = 0
+    until [1, 2].include?(decision)
+      puts 'Do you want to create a student (1), or a teacher (2)?'
+      print INPT_MSG
+      decision = gets.chomp.to_i
+      next if [1, 2].include?(decision)
+
+      puts
+      puts 'Invalid input, please try again'
+      puts
+    end
+    print 'Age --> '
+    age = gets.chomp.to_i
+    print 'Name --> '
+    name = gets.chomp
+    case decision
+    when 1
+      create_student(age, name)
+    else
+      create_teacher(age, name)
+    end
+    puts
+  end
```

Comment:
From the method name `create_person`, i should predict its responsible for : 
 - creating a person (student / teacher) instance.

But actually what its doing are : :thinking: 
 - line 28 to 37 : validate (and getting) user input.
 - lines 39, 41 : asking for user inputs

So what we need to do here (as requested in the project [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_refactor.md#:~:text=pay%20special%20attention%20to%20the%20%22single-responsiblity%20principle%22.%20)) is to make sure our method more simple :1234: 
-  follow the [single-responsiblity principle](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid)
- all requests for input and input validation should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input.

Suggestion on how :question: 
- Extract those code snippets from your `app` module methods and put it in separate methods at `main` file.
- Then pass any inputs needed to your `app` module